candi/control-plane-kubeadm/config.yaml.tpl

```diff
@@ -159,12 +160,10 @@ scheduler:
     config: "/etc/kubernetes/deckhouse/extra-files/scheduler-config.yaml"
 {{- end }}
     profiling: "false"
-{{- if semverCompare ">= 1.21" .clusterConfiguration.kubernetesVersion }}
-    feature-gates: "EndpointSliceTerminatingCondition=true,DaemonSetUpdateSurge=true"
-{{- end }}
 {{- if semverCompare "< 1.20" .clusterConfiguration.kubernetesVersion }}
-    feature-gates: "DefaultPodTopologySpread=true"
+    {{- $featureGates = "DefaultPodTopologySpread=true" }}
```

Comment:
I believe it's better to leave this part as it was before. This override of `featureGates` variable makes no sense to me.